powertools-validation/src/main/java/software/amazon/lambda/powertools/validation/ValidationConfig.java

```diff
@@ -85,8 +85,13 @@ public void setSchemaVersion(SpecVersion.VersionFlag version) {
      * @param <T> Must extends {@link BaseFunction}
      */
     public <T extends BaseFunction> void addFunction(T function) {
-        configuration.functionRegistry().extend(function);
-        jmesPath = new JacksonRuntime(configuration, getObjectMapper());
+        FunctionRegistry functionRegistryWithExtendedFunctions = configuration.functionRegistry().extend(function);
```

Comment:
This was a neat finding and was actually a bug

Reply:
👍